bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -89,51 +80,17 @@ public String toString() {
         }
     }
 
-
-    public enum AnalyseChoice {
-        BOTH("Canopus and GSAOI", GemsTipTiltMode.both),
-        CANOPUS("Canopus", GemsTipTiltMode.canopus),
-        GSAOI("GSAOI", GemsTipTiltMode.instrument),
-        ;
-
-        public static AnalyseChoice DEFAULT = CANOPUS; // REL-604
-
-        private String _displayValue;
-        private GemsTipTiltMode _gemsTipTiltMode;
-
-        AnalyseChoice(String name, GemsTipTiltMode gemsTipTiltMode) {
-            _displayValue = name;
-            _gemsTipTiltMode = gemsTipTiltMode;
-        }
-
-        public String displayValue() {
-            return _displayValue;
-        }
-
-        public String toString() {
-            return displayValue();
-        }
-
-        public GemsTipTiltMode getGemsTipTiltMode() {
-            return _gemsTipTiltMode;
-        }
-    }
-
     private final GemsInstrument instrument;
-    private final GemsTipTiltMode tipTiltMode;
-    private final Set<Angle> posAngles;
+    private final Set<Angle>     posAngles;
+
     public static final CatalogChoice DEFAULT = CatalogChoice.DEFAULT;
 
-    public GemsGuideStarSearchOptions(final GemsInstrument instrument,
-                                      final GemsTipTiltMode tipTiltMode, final Set<Angle> posAngles) {
+    public GemsGuideStarSearchOptions(
+        final GemsInstrument instrument,
+        final Set<Angle>     posAngles
+    ) {
```

Comment:
Here we need to include an option about _whether_ to do the flexure star search or not.  By default, we don't want to do it and we definitely don't want to fail the search if we can't find an ODGW star.  If not doing flexure star search, there should be no `instrumentCriterion`.

Reply:
I believe the flexure star search _should_ be disentangled from the asterism/tip-tilt search but it appears to be difficult.  See `GemsResultsAnalyzer`.